src/unity/python/turicreate/visualization/plot.py

```diff
@@ -0,0 +1,29 @@
+import time
```

Comment:
These imports will get exported publicly from this file - prefix them with underscores like `import time as _time` to prevent that.

Reply:
Noted changed the public imports to _imports